input/pagecontent/reference.md

```diff
@@ -697,52 +703,127 @@ FSH provides a shortcut for setting several of the most common properties of a C
   * type = urn:iso-astm:E1762-95:2013#1.2.840.10065.1.12.1.2 "Coauthor's Signature"
   ```
 
+* What happens when a second assignment replaces an existing value:
+
+  ```
+  * myCoding = SCT#363346000 "Malignant neoplastic disease (disorder)"
+  * myCoding = ICD10#C80.1
+  
+  // Result:
+  //   myCoding.system is ICD10
+  //   myCoding.code is C80.1
+  //   myCoding.display does not exist
+  //   myCoding.version does not exist
```

Comment:
I'm probably just being picky so feel free to ignore me, but "does not exist" felt odd to me. I think something like "has no value" or "is not set to a value" feels more descriptive.

Reply:
changed to "has no value"